### PR TITLE
Fix some linter warnings in settings

### PR DESCRIPTION
**Pull request by rullzer**
It is not all but at least less warnings now.

### CODE REVIEW

File: apps/settings/src/components/AdminTwoFactor.vue [hunk 1]
```diff
@@ -69,6 +69,7 @@
 import axios from '@nextcloud/axios'
 import { Multiselect } from '@nextcloud/vue'
 import _ from 'lodash'
+import { generateUrl, generateOcsUrl } from "@nextcloud/router"
```
Comment (MorrisJobke) [inline apps/settings/src/components/AdminTwoFactor.vue:72]: /home/runner/work/server/server/apps/settings/src/components/AdminTwoFactor.vue
12900
      72:45  error  Strings must use singlequote  quotes
12901
    
12902
    ✖ 1 problem (1 error, 0 warnings)
12903
      1 error and 0 warnings potentially fixable with the `--fix` option.

Comment (MorrisJobke) [inline apps/settings/src/components/AdminTwoFactor.vue:72]:
```suggestion
import { generateUrl, generateOcsUrl } from '@nextcloud/router'
```

Comment (rullzer) [reply apps/settings/src/components/AdminTwoFactor.vue:72]: O joy... so my phpstorms autocomplete uses double quotes

**Review by skjnldsv — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**